test/data/testSPun.py

```diff
@@ -168,6 +163,24 @@ def testmosyms(self):
         self.assertEquals(shape1, (2, self.data.homos[1]+11))
 
 
+class MolcasSPunTest(GenericSPunTest):
+    """Customized unrestricted single point unittest"""
+
+    # In MOLCAS we are parsing not only the alpha/beta orbitals but also
+    # the Natural orbitals. Hence we have 3 homos.
+    def testhomos(self):
+        """Are the homos correct?"""
+        msg = "%s != array([34,33,40],'i')" % numpy.array_repr(self.data.homos)
+        numpy.testing.assert_array_equal(self.data.homos, numpy.array([34,33,40],"i"), msg)

```

Comment:
Discussed in a previous comment that this should be `[34, 33]`, which means that `MolcasSPunTest` is unnecessary.